app/controllers/user_controller.rb

```diff
@@ -185,6 +185,25 @@ def signup
           end
         end
 
+        # Prevent signups from spam domains
+        if UserSpamScorer.new.email_from_spam_domain?(@user_signup)
+          if send_exception_notifications?
+            msg = "Attempted signup from spam domain email: " \
+                  " #{ @user_signup.email }"
+            e = Exception.new(msg)
+            ExceptionNotifier.notify_exception(e, :env => request.env)
+          end
+
+          if AlaveteliConfiguration.enable_anti_spam
+            flash.now[:error] =
+              _("Sorry, we're currently unable to sign up new users, " \
```

Comment:
I think we should keep it as-is for now. We haven't got anti-spam on for most sites, so no one will really see this unless we're dealing with an influx.